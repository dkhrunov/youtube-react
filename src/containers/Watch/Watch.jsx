import React from 'react';
import VideoMetadata from '../../components/VideoMetadata/VideoMetadata';
import RelatedVideos from '../../components/RelatedVideos/RelatedVideos';
import Video from '../../components/Video/Video';
import './Watch.scss';
import VideoInfoBox from '../../components/VideoInfoBox/VideoInfoBox';

const Watch = (props) => {
	return (
		<div className='watch-grid'>
			<Video className='video' id='-7fuHEEmEjs' />
			<VideoMetadata className='metadata' viewCount={1000} />
			<VideoInfoBox className='video-info-box' />
			<div
				className='comments'
				style={{ width: '100%', height: '100px', background: '#9013FE' }}
			>
				comments
			</div>
			<RelatedVideos className='relatedVideos' />
		</div>
	);
};

export default Watch;
