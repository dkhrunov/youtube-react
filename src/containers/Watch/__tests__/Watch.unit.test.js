import React from 'react';
import { shallow } from 'enzyme';
import Watch from '../Watch';

describe('Renders NextUpVideo component', () => {
	test('renders', () => {
		const wrapper = shallow(<Watch />);
		expect(wrapper).toMatchSnapshot();
	});
});
