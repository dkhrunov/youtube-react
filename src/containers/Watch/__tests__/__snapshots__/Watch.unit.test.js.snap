// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NextUpVideo component renders 1`] = `
<div
  className="watch-grid"
>
  <Video
    className="video"
    id="-7fuHEEmEjs"
  />
  <VideoMetadata
    className="metadata"
    viewCount={1000}
  />
  <VideoInfoBox
    className="video-info-box"
  />
  <div
    className="comments"
    style={
      Object {
        "background": "#9013FE",
        "height": "100px",
        "width": "100%",
      }
    }
  >
    comments
  </div>
  <RelatedVideos
    className="relatedVideos"
  />
</div>
`;
